packages/nextjs/src/api/index.ts

```diff
@@ -35,4 +43,6 @@ export {
   organizations,
   clients,
   allowlistIdentifiers,
+  createClerkClient,
+  clerkClient,
```

Comment:
❓ Based on the title i was expecting the addition of some types in the export. I see that we have also exported the  `clerkClient`. Is it intentional?